internal/core/data/init.go

```diff
@@ -154,5 +152,12 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, wg *sync.WaitGroup, st
 		},
 	})
 
+	if configuration.Databases["Primary"].Type == "redisdb" {
```

Comment:
@cloudxxx8 , Code checks like this is why these functions need to be part of the App layer. The will be needed for other DB implementations.